tsl/test/sql/dist_triggers.sql

```diff
@@ -193,7 +185,29 @@ FROM trigger_events
 GROUP BY 1,2,3,4
 ORDER BY 1,2,3,4;
 
-SELECT * FROM test.remote_exec(ARRAY[:'DATA_NODE_1', :'DATA_NODE_2', :'DATA_NODE_3'], $$
+SELECT * FROM test.remote_exec(ARRAY[:'DATA_NODE_1', :'DATA_NODE_2'], $$
+SELECT tg_when, tg_level, tg_op, tg_name, count(*)
+FROM trigger_events
+GROUP BY 1,2,3,4
+ORDER BY 1,2,3,4;
+$$);
+
+-- Attach a new data node and show that the hypertable is created on
+-- the node, including its triggers.
+SELECT attach_data_node(:'DATA_NODE_3', 'hyper');
```

Comment:
What happens if `trigger_events` do not exist on this data node? You propagate the trigger function, but not any associated tables (AFAICT). This should result in an error when the trigger is created, but I see no test for it.

Reply:
No, there will be no error when the trigger is created since the function isn't introspected at that time. The failure happens when the function is executed. I didn't test this case since it is the standard behavior of functions to fail when executed if they try to access objects that don't exist. Not sure why we need to test that standard behavior here, but I will nevertheless add this case since it is easy to do.

Reply:
Hmmm... I got an error before when trying to create a function referencing a table but cannot reproduce it. Guess I remember it wrong.